### DISCUSSION TITLE
Chocolatey Package Folder

### DISCUSSION THREAD

**Discussion by Manoj496** [chocolatey/choco#2683] [General]
Hi,

By default chocolatey package folders are installed/updated to C:/Programdata/chocolatey/lib. Right now we've a default destination path where our scripts are installed and using Get-packageparamaters, we modified installation script in such a way that when using command line (choco install <packageName> -d --package-parameters "'/InstallationPath:""C:\temp\folder'") it will install only to the InstallationPath given in commandline. In this scenario the package folder in "C:/Programdata/chocolatey/lib" also gets updated. Is there any option not to update the packagefolder when using the commandline. As we don't want $env:ChocolateyPackageFolder to g et updated.

**Comment by pauby** [comment]
Choco needs to update the lib folder as that's where it keeps the package files.

What updates do you not want to happen in there?

**Comment by Manoj496** [comment]
Like I said, when installing the package using commandline like I stated above, scripts from package will be installed and lib folder also get updated simultaneously.  Is there any way to stop updating when using commandline or can we create another environment to do so. 
Eventually those lib folder should updated only when installing the files to default destination path.

**Reply by pauby** [reply]
Chocolatey uses that lib folder to keep the package files in there. There is no way to change the behaviour of updating the lib folder by Chocolatey. If the package contains binary files then you can remove those as part if the package does. But don't remove the standard Chocolatey files.

You can change where the software is installed, like you are doing. That's separate from the package itself.
